src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSidebar.tsx

```diff
@@ -52,15 +52,17 @@ export const PanelSideBar = <TPanelItemId extends string, TPanelItem>() => {
 
     const PanelItemsRenderer = (props: { items: PanelItem<TPanelItemId, TPanelItem>[] }) => {
       const { items } = props;
-      return items?.map((item, index) =>
-        renderTilesAsLinks ? (
-          <LinkRenderer key={index} item={item}>
-            <ButtonIcon item={item} />
-          </LinkRenderer>
-        ) : (
-          <ButtonIcon key={index} item={item} />
-        ),
-      );
+      return items
+        ?.filter((x) => x.display)
```

Comment:
Can you check only for case in which display is explicitely false? So that we don't have to provide true as default value.

x.display !== false